ASP.NET Core/gulpfile.js

```diff
@@ -0,0 +1,62 @@
+/// <binding Clean='clean' />
+var gulp = require("gulp"),
+    rimraf = require("rimraf"),
+    concat = require("gulp-concat");
+
+var paths = {
+    webroot: "./wwwroot/"
+};
+
+paths.concatJsDest = paths.webroot + "js/vendor.js";
+paths.concatCssDest = paths.webroot + "css/vendor.css";
+paths.fonts = paths.webroot + "css/fonts";
+paths.icons = paths.webroot + "css/icons";
+
+let scripts = ["node_modules/jquery/dist/jquery.js",
+    "node_modules/bootstrap/dist/js/bootstrap.js",
+    "node_modules/devextreme/dist/js/dx.all.js",
+    "node_modules/devextreme/aspnet.js",
+    "node_modules/devextreme-aspnet-data/js/dx.aspnet.data.js"];
+
+let styles = [
+    "node_modules/bootstrap/dist/css/bootstrap.css",
+    "node_modules/devextreme/dist/css/dx.common.css",
+    "node_modules/devextreme/dist/css/dx.material.blue.light.compact.css"];
```

Comment:
Also, I think it is a good practice to add site.css and site.js to the resulting bundle. What do you think?

Reply:
decided not to touch